stl_file_path : "resources/petbottle_rectangle.stl"
frame_id : "world"
init_x_position : 1.0
init_y_position : 1.0
init_z_position : 1.0
x_scale : 0.001
y_scale : 0.001
z_scale : 0.001
